openmc/opencg_compatible.py

```diff
@@ -901,7 +901,9 @@ def get_opencg_lattice(openmc_lattice):
 
     # Convert 2D universes array to 3D for OpenCG
     if len(universes.shape) == 2:
-        universes.shape = (1,) + universes.shape
+        new_universes = universes.copy()
+        new_universes.shape = (1,) + universes.shape
+        universes = new_universes
```

Comment:
You are correct! When I was updating this last night, I discovered that one of the cells in the Pandas DataFrame Jupyter Notebook example failed when building a DF for a distribcell tally using summary info from OpenCG. In particular, the code here turned the `universes` for a 2D lattice was converted into a 3D NumPy array, while leaving all of other lattice attributes such as `dimension`, `pitch` and `lower_left` as 2D.

This bug was related to PR #539 (and perhaps #534), although I found it strange since I separately ran all Notebooks while reviewing both PRs. I can submit a separate PR specifically for this bug fix, or can leave it here if we want to move this PR into the code quickly.